tests/conftest.py

```diff
@@ -139,6 +139,14 @@ def test_session(crmp_session, caplog):
     yield crmp_session
 
 
+@pytest.yield_fixture(scope='function')
+def test_path_exception(test_session):
```

Comment:
It's usually more clear to name test fixtures with the name of a noun, since it's a *thing* that you'll be using in your test. E.g. `session_with_user`.